src/App.js

```diff
@@ -40,29 +48,29 @@ class App extends Component  {
     // PokeAPI(this.state.offset)
   }
   
-  // unFavoritePokemon = (id) => {
-  //   // unfavoriting
-  //   //find the fav pokemon from the fav pokemon array and remove it 
-  //   // and store the new array to a variable name
-  //   //unfavoritng button needed
-  //   const unpokemon = this.state.favoritePokemon.splice((id, 1))
-  //   // same thing in line 34
-  //   console.log(unpokemon)
-  //   this.setState({ ...this.state, favoritePokemon: unpokemon })
-  // }
+  deletePokemon = (pokename) => {
+    console.log(pokename)
+    const findme = this.state.favoritePokemon.filter(findmepls => pokename !== findmepls.name)
+    console.log(findme)
+    this.setState({ favoritePokemon: findme})
+    // unfavoriting
+    //find the fav pokemon from the fav pokemon array and remove it 
+    // and store the new array to a variable name
+    // const unpokemon = this.state.favoritePokemon.splice((id, 1))
+    // this.setState({ ...this.state, favoritePokemon: unpokemon })
+  }
   
   
   render() {
-    console.log(this.state.favoritePokemon)
     return(
     <div className="App">
         <NavComponent />
       <header className="App-header">
         <Switch>
           <Route exact path='/' render={() => <iframe className='poke-trailer' title="Pokemon Song" width="560" height="315" src="https://www.youtube.com/embed/rg6CiPI6h2g?autoplay=1" ></iframe>} />
```

Comment:
You could consider playing a little bit with these lines to make them more readable, but that's not mission critical, just preference.